ShittyLINQ/All.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace ShittyLINQ
+{
+    public static partial class Extensions
+    {
+        /// <summary>
+        /// True represents that each item satisfied the given condition.
+        /// </summary>
+        public static bool All<T>(this IEnumerable<T> self, Func<T, bool> predicate)
+        {
+            if (predicate == null)
+            {
+                throw new ArgumentNullException("The predicate is required.");
+            }
+
+            Func<bool, T, bool> function = (memo, obj) =>
+            {
+                memo &= predicate(obj);
```

Comment:
ShittyLINQ is such a beautiful thing. 
I legitimately was not aware of the `&=` operator.